Martin-Hinshelwood/Martin-Hinshelwood.md

```diff
@@ -7,7 +7,7 @@ qualifications:  "BSc Computer Studies, Microsoft Certified Professional, Micros
 quote:  "“Program testing can be used to show the presence of bugs, but never to show their absence!”"
 ---
 
-Martin is a Solution Architect at SSW, and also one of three [Microsoft Visual Studio ALM MVP](https://mvp.support.microsoft.com/profile/Martin.Hinshelwood) in the UK and has over 9 years experience in the software industry. He regularly writes on his [ALM blog](https://blog.hinshelwood.com/), has many articles published on [Code Project](https://www.codeproject.com/Members/Martin-Hinshelwood) and is a moderator on Microsoft forums. His expertise includes (but is not exclusive to) Team Foundation Server, Scrum, WPF, Silverlight, SharePoint, VB.NET, C#, WCF, ASP.NET. 
+Martin is a Solution Architect at SSW, and also one of three [Microsoft Visual Studio ALM MVP](https://mvp.support.microsoft.com/profile/Martin.Hinshelwood) in the UK and has over 9 years experience in the software industry. He regularly writes on his [ALM blog](https://blog.hinshelwood.com/), has many articles published on [Code Project](https://www.codeproject.com) and is a moderator on Microsoft forums. His expertise includes (but is not exclusive to) Team Foundation Server, Scrum, WPF, Silverlight, SharePoint, VB.NET, C#, WCF, ASP.NET. 
```

Comment:
https://blog.hinshelwood.com/

This link is broken

Reply:
Done - updated URL